### ISSUE TITLE
New Buttons on Etcd Backup Page

### ISSUE THREAD

**Issue by luthermonson** [rancher/rancher#25511]
Navigate to a cluster and the main /c/<id>/monitoring page has a ... menu which contains a lot of utilities. Two are "snapshot now" and "restore a snapshot". If you go to Tools > Sanpshots there are no buttons and could be useful to add "Restore a Snapshot" to the ... menu for a single snapshot and add a "Snapshot Now" at the top of the page. Functionality should be the same as the existing buttons, just nicer to have these features in the main snapshot area.

**Comment by deniseschannon** [comment]
Merged into `master` before 2.4 shipped, so available in v2.4-head and master-head

**Comment by izaac** [comment]
Rancher version: `v2.4.2` 
Rancher version: `v2.4-2700-head (04/06/2020)` commit id: 481cb64b3

I tested the backup/restore with `etcd, Kubernetes Version and cluster config` and `etcd and Kubernetes Version` options from the new UI elements in Cluster -> Tools -> Snapshots UI.

Looks good, I still got to cover `master-head` after #26508 is fixed.

**Comment by izaac** [comment]
Rancher version: `master-2720-head (04/07/2020)` commit id: 3c6eb2b74

I tested the backup/restore with `etcd, Kubernetes Version and cluster config` and `etcd and Kubernetes Version` options from the new UI elements in Cluster -> Tools -> Snapshots UI.
